### DISCUSSION TITLE
Searchable Dropdown Select

### DISCUSSION THREAD

**Discussion by supermx1** [Budibase/budibase#2305] [Ideas]
So, was playing around with budibase and realised that there is no option to activate search/filter on a dropdown/picker.... Weird. I have a lot of M2O and O2O relationships that require a search or filter. Like for instance a table has over 1000 records and is connect in a relationship to another table. I have to scroll through hundreds of records to make a selection. I don't know if i am missing something or is there a menu option we are not seeing?

**Comment by joebudi** [comment]
This is something that is not possible right now. 

We've chatted internally about adding a Combobox. This is a valid request and a great idea. I am going to change the category to Idea so we don't miss it in our planning iteration meeting.

**Reply by tobehn** [reply]
Combobox surely is the way to go.
#2300 #2316

**Comment by berma83** [comment]
+1 

We need this feature ASAP. Thank you!

**Comment by crudyrudy** [comment]
Indispensable for many applications I would say

**Reply by tobehn** [reply]
There has work been done already see #2372.
I looks like there has even a Pull Request been accepted. Though i cannot find any information about the release.
@aptkingston could you give an update on when this feature might be released? Thank you so much in advance.
